vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.h

```diff
@@ -30,7 +30,8 @@ class MediaTwitter : public ledger::LedgerCallbackHandler {
   void SaveMediaInfo(const std::map<std::string, std::string>& data,
                      ledger::PublisherInfoCallback callback);
 
-  std::string GetShareURL(const std::map<std::string, std::string>& args);
+  static std::string GetShareURL(
```

Comment:
Maybe I am missing something, but I don't see change in `bat_get_media` that would reflect this static change

Reply:
Good catch, fixed.